src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -491,4 +526,41 @@ public String toString() {
       return name().toLowerCase(Locale.ROOT);
     }
   }
+
+  /**
+   * Enum which represents the type of processing guarantees that the customer want (either
+   * at_least_once (default) or exactly_once
+   */
+  public enum ProcessingGuarantee {
```

Comment:
Should this be `IngestionProcessingGuarantee`

Reply:
Yah, I can update the naming